tools/generate-wire.py

```diff
@@ -126,6 +126,16 @@ def __init__(self, message, name, size, comments, prevname):
         self.lenvar = None
         self.num_elems = 1
         self.optional = False
+        self.is_tlv = False
+
+        # field name appended with '+' means this field contains a tlv
+        if name.endswith('+'):
```

Comment:
This isn't going to work well in the spec; let's use name.endswith('tlv')?

Reply:
updated!!